### ISSUE TITLE
extension not working

### ISSUE THREAD

**Issue by LaveshChaudhari** [mhutchie/vscode-git-graph#818]
Bug Description:
In VsCode git graph extension when i try to open the extension it is not opening. i tried to enable and disable the extension multiple times but still getting the same issue . even after uninstalling and installing the extension several times again still getting the same issue at least provide a good understandable error message if something is wrong from my side .

Steps to reproduce the behaviour:
1. Go to 'Git graph extension open it'
2. see error message : " Error loading webview: Error: Could not register service worker: InvalidStateError: Failed to register a ServiceWorker: The document is in an invalid state.."

**Expected Behaviour**:
should open after enabling and disabling multiple times

**Environment**
 - Git Graph Extension Version: 1.13.0
 - Visual Studio Code Version:  1.37.1
 - Operating System: macOs

⚠ Please make sure you complete all of the required sections of this template. Without this required information, it will be harder to replicate this bug, and cause additional delays in resolving it.

**Comment by ssteinerx** [comment]
I have been having this issue on and off for a while.  

Whatever causes it also seems to leave orphan VSCode code chunklets running that will make sure it doesn't work and also helpfully leaves behind dysfunctional cached code chunks that will make sure it will continue to not work, even if you reboot!

In Ubuntu, this fixes it for me.  I don't use Windows or OS X. 

- NOTE: I'm using the `code-insider` version.  If you're using just `code`, use that below, but make sure you're not killing other `code` things that aren't VSCode things.

- NOTE: You could just reboot and GOTO __No Leftovers__, but seeing the orphan chunklets confirms that the problem is the same-ish as the one that appears to be causing the issue for me.
 
- Quit VSCode --  all the way out.

__Check for Leftovers__:

```bash
$ pgrep -l code-insider
612285 code-insiders
612294 code-insiders
...and so forth, on and on.
```
-  If there are none, GOTO __No Leftovers__.

- Kill them all:

`$ killall code-insiders`

- If you've tried a couple of times and they just won't die:

`$ sudo killall code-insiders`

- If  `sudo killall` _still_ didn't kill them still...reboot.  Desperate times and all that.  GOTO __No Leftovers__.

GOTO __Check for Leftovers__.

__No Leftovers__:

- Remove the webbish cache with the broken chunks:

`$ rm -r ~/.config/Code/CachedData`

- Open VSCode, go to Git (`Ctrl-Shift-G`), press the *Git Graph* icon and marvel at the beauty that is *Git Graph*.

- Keep Calm and Carry On.

END.

**Comment by clegendre** [comment]
Thank you!
Happened to me today, on Windows:
Cache location:
` ~\AppData\Roaming\Code\CachedData`

**Comment by ssteinerx** [comment]
> Thank you! Happened to me today, on Windows: Cache location: ` ~\AppData\Roaming\Code\CachedData`

And killing the stray processes and removing the cached data worked on Windows?

**Comment by clegendre** [comment]
> > Thank you! Happened to me today, on Windows: Cache location: ` ~\AppData\Roaming\Code\CachedData`
> 
> And killing the stray processes and removing the cached data worked on Windows?

Yep, killing all code processes first, then removing the cached data, worked on Windows 👍

**Comment by ssteinerx** [comment]
@mhutchie Would you like a pull request to add a markdown file with these instructions tweaked up for Linux and Windows if @clegendre could help with that part?

I don't use OS X any more, so, even though it's pretty Linux compatible-ish, I can't test the instructions there.
